Documentation/guides/messages/apt0001.md

```diff
@@ -8,7 +8,7 @@ ms.date: 08/01/2018
 ## Example messages
 
 ```
-error APT0001: unknown option -- --no-crunch . This is the result of using `aapt` command line arguments with `aapt2`. The arguments are not compatible.
+error APT0001: Unknown option `--no-crunch`. Please check `$(AndroidAapt2CompileExtraArgs)` and `$(AndroidAapt2CompileExtraArgs)` to see if they include any `aapt` command line arguments that are no longer valid for `aapt2` and ensure that all other arguments are valid for `aapt2`.
```

Comment:
The only issue with this error message, is `--no-crunch` seems like it is an option for `aapt2`:
```
> .\bin\Debug\lib\xamarin.android\xbuild\Xamarin\Android\aapt2.exe compile --help
aapt2 compile [options] -o arg files...

Options:
 -o arg                                            Output path
 --dir arg                                         Directory to scan for resources
 --zip arg                                         Zip file containing the res directory to scan for resources
 --output-text-symbols arg                         Generates a text file containing the resource symbols in the
                                                   specified file
 --pseudo-localize                                 Generate resources for pseudo-locales (en-XA and ar-XB)
 --no-crunch                                       Disables PNG processing
 --legacy                                          Treat errors that used to be valid in AAPT as warnings
 --visibility arg                                  Sets the visibility of the compiled resources to the specified
                                                   level. Accepted levels: public, private, default
 -v                                                Enables verbose logging
 --trace-folder arg                                Generate systrace json trace fragment to specified folder.
 -h                                                Displays this help menu
```
Maybe they added this in a recent version of aapt2?

Should we just use `--banana` as an example or something?

Reply:
:+1 for `--banana`